broker/broker-app/src/main/resources/features.xml

```diff
@@ -87,15 +88,15 @@
         <bundle>mvn:org.apache.activemq/artemis-server-osgi/${artemis.version}</bundle>
     </feature>
 
-    <!-- This feature has a start level of 80 to keep it higher than the "camel-amqp" feature
-        The "camel-amqp" feature uses a different version of proton than this feature. If this version of
-        proton starts before the "camel-amqp" version, that introduces version conflict issues.
-
-        TODO: DDF-3029 Once we have upgraded to Artemis 2.1.0, both "artemis-amqp" and "camel-amqp" should
-        use the same version of proton and the explicit start level on this feature should not be necessary.
-        -->
+    <!--
+        If the "camel-amqp" feature uses a different version of proton than this feature, version
+        conflicts will be introduced if this feature is started first. Adding a start order with a
```

Comment:
✏️  "Adding a start-level of 80"?

Also, the last "camel-amqp" is missing a closing quotation mark.